packages/core/src/components/ui/Logo/Logo.tsx

```diff
@@ -1,5 +1,26 @@
-function Logo() {
-  return <span data-fs-logo role="img" aria-label="BaseStore logo" />
+import { Image } from 'src/components/ui/Image'
+import type { ImageProps } from 'src/components/ui/Image'
+
+interface LogoProps {
+  alt: string
+  src: string
+  loading?: ImageProps['loading']
+}
+
+function Logo({ alt, src, loading = 'lazy' }: LogoProps) {
+  return (
+    <div data-fs-logo>
+      <Image
+        alt={alt}
+        src={src}
+        width={0}
+        height={0}
+        sizes="100vw"
+        loading={loading}
+        style={{ width: '100%', height: 'auto' }}
+      />
```

Comment:
I also liked this approach making the logo discoverable using Image instead of import from CSS, this can be good for performance. 

Regarding:
```js
width={0}
height={0}
sizes="100vw"
```

Is it working as expected? `width` and `height` equals `0` may worse the CLS (Cumulative Layout Shift). 
`sizes="100vw"` represents that this logo fill all the screen width and will make Next Image download a huge image degrading performance. We could use other configs (take a look at other image components).